Reqnroll/EnvironmentAccess/BuildMetadata.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace Reqnroll.EnvironmentAccess
```

Comment:
The PR looks good, I'm only doubting about the namespace. I find "access" a bit unclear.

maybe BuildEnvironment?

Reply:
The Namespace was one that already existed (it contains the Environment variable access service and the Environment Info access service. I added this service to that namespace (and corresponding directory in the folder structure).

Do you feel that this new service is better suited elsewhere in the project structure? Should it have its own folder/namespace?

Reply:
I missed that is was already there. Then I think it's OK, a separate namespace is not needed IMO

Thanks!